run-cnf-suites.sh

```diff
@@ -126,3 +127,8 @@ else
 fi
 
 cd ./cnf-certification-test && ./cnf-certification-test.test $FOCUS_STRING $SKIP_STRING $LABEL_STRING ${GINKGO_ARGS}
+
+if [ -d ${TNF_PARTNER_DIR}/src ] &&  [[ ! -z "${TNF_PARTNER_SRC_DIR}" ]]; then
```

Comment:
I don't think we need this extra check. I believe we should remove these two env vars [from the github workflow](https://github.com/test-network-function/cnf-certification-test/blob/cad0513bbe1103b5fac5a2301d928d0f34c90121/.github/workflows/pre-main.yaml#L23). Both env vars (`TNF_PARTNER_DIR` and `TNF_PARNER_SRC_DIR`) are used [by the tnf Dockerfile](https://github.com/test-network-function/cnf-certification-test/blob/cad0513bbe1103b5fac5a2301d928d0f34c90121/Dockerfile#L2) to download the partner repo into a specific container folder. Thus, `TNF_PARNER_SRC_DIR` is checked because when `run-cnf-suites.sh` is running from the container, we want to call the script to create the daemonset.
So, if I'm not wrong, [your original commit](https://github.com/test-network-function/cnf-certification-test/commit/9b0fbd5c83024defb46898de3195f323b708ef37) will work well if those two env vars are removed from the workflow parameters.
Other than that, I don't know why there's two env vars... just `TNF_PARTNER_DIR` would do it, but that could be addressed in a different PR.

Reply:
Okay I have removed the ENV vars.